config/fam_pe_config.yaml

```diff
@@ -0,0 +1,23 @@
+# PE config file
```

Comment:
Add more explanations to all the options : 
Like : 
including explanation for all available values for parameters
default value